css/pages/discord.css

```diff
@@ -1,39 +1,81 @@
-.main-heading{
+.discord-wrapper {
+  margin-left: 5%;
+}
+
+.main-heading {
+
   text-align: left;
   font-weight: 600;
-  padding: 10px;
+
+  margin: 40px 0 30px 0;
 }
-body{
-    margin-left: 5%;
+
+.get-started {
+  text-align: left;
+  font-weight: 550;
+  
+}
+.discord > *{
+  text-align: left;
+  font-weight: 500;
+  /* margin-left: 2%; */
+  font-size: 1em;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;  
+
 }
 
-.discord-info{
+.discord-info {
+  
   text-align: left;
   font-weight: 500;
-  margin-left: 2%;
+  /* margin-left: 2%; */
+  font-size: 1em;
+  margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+}
 
+div {
+  margin-top: 30px;
+  margin-bottom: 40px;
 }
-.get-started{
+
+.category-img{
+  margin-left: 2px;
+  width: 61%;
+}
+
+ul {
   text-align: left;
-  font-weight: 550;
+  font-weight: 500;
+  margin-left: 0.5%;
+  padding: 10px;
+}
+ol {
+  margin-left: 4%;
+  font-weight: 500;
   padding: 10px;
 }
 
-ul{
-    text-align: left;
-    font-weight: 500;
-    margin-left: 2%;
-    padding:10px
-    
+.category-name {
+  margin-top: 10px;
+}
+.channel-name {
+  margin-top: 5px;
 }
 
-ol{
-   margin-left:4%;
-   font-weight: 500;
-   padding: 10px;
+@media only screen and (max-width: 650px) {
+  .category-img {
+    width: 90%;
+  }
 }
 
-.category-img{
-    display: block;
-    margin-left: 2%;
-}
\ No newline at end of file
+@media only screen and (min-width: 651px) and (max-width: 950px) {
+  .category-img {
+    width: 85%;
+  }
+}
```

Comment:
indentation looks better this time. Are you using any pluggin to manage indents and punctuations?

Reply:
I have used the prettier code formatter the one you had suggested.